mne/io/snirf/_snirf.py

```diff
@@ -365,7 +365,8 @@ def natural_keys(text):
                     data = np.atleast_2d(np.array(
                         dat.get('/nirs/' + key + '/data')))
                     if data.size > 0:
-                        annot.append(data[:, 0], 1.0, key[4:])
+                        desc = dat.get('/nirs/' + key + '/name')[0]
```

Comment:
For anyone else reviewing, this corresponds to https://github.com/fNIRS/snirf/blob/master/snirf_specification.md#nirsistimjname